plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
```

Comment:
We might want to take a look at this tomorrow together, but beware the code executed in each entry of the container will be affecting only  the `InputSpec` that is perfectly fine, because it's just about collecting the commands to be used later in the `Input` task